src/components/layout/header.vue

```diff
@@ -1,26 +1,38 @@
 <template>
-  <div class="header">
-    <icons-IconLogoBlue/>
-    <input type="text" class="header_vk_id" placeholder="Поиск" />
-  </div>
+  <slot name="header">
+    <img src="https://cdn.icon-icons.com/icons2/1392/PNG/512/cart_96642.png" alt="Корзина" width="30" height="30"/>
+    <span class="cart-badge">{{ cartCount}}</span>
+  </slot>
```

Comment:
You shouldn;t delete old template. You should add <slot /> inside old template and then use it on page as 
```
<layout-header><cart-header /></layout-header>
```

Reply:
check docs https://vuejs.org/guide/components/slots.html

Reply:
place  `<slot />` to the right side of header